composer.json

```diff
@@ -13,6 +13,6 @@
         "psr-4": { "MangoPay\\": "MangoPay/" }
     },
     "require-dev": {
-        "simpletest/simpletest": "^1.1"
+        "phpunit/phpunit": "^4"
```

Comment:
Should be `4.8.36` to ensure you're actually receiving a version with the forward compat layer (namespaced classes)